lib/samples/apply_class_breaks_renderer_to_sublayer/apply_class_breaks_renderer_to_sublayer_sample.dart

```diff
@@ -28,17 +28,22 @@ class ApplyClassBreaksRendererToSublayerSample extends StatefulWidget {
 class _ApplyClassBreaksRendererToSublayerSampleState
     extends State<ApplyClassBreaksRendererToSublayerSample> {
   final _mapViewController = ArcGISMapView.createController();
-  // create a map with a basemap
+  // create a map with a basemap style
   final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISStreets);
   // set the rendered state to false
   bool _rendered = false;
   // set the ready state to false
   bool _ready = false;
+  // create an image sublayer
+  late ArcGISMapImageSublayer countiesSublayer;
+  // create an renderer
+  late Renderer defaultRenderer;
```

Comment:
I think generally we've been naming all member variables with a leading `_`. So that should be `_countiesSublayer`, `_defaultRenderer`, and also `_map` a couple lines up.